releasenotes/notes/deprecate-pulse-Call-instruction-538802d8fad7e257.yaml

```
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    The :class:`~qiskit.pulse.instructions.Call` has been deprecated and will
+    be removed in a future release.
+    Instead, use the pulse builder function :func:`~qiskit.pulse.builder.call`
+    within an active building context.
```

```suggestion
deprecations:
  - |
    The :class:`~qiskit.pulse.instructions.Call` has been deprecated and will
    be removed in a future release.
    Instead, use the `pulse builder
     <https://qiskit.org/documentation/tutorials/circuits_advanced/06_building_pulse_schedules.html>`_ 
    function :func:`~qiskit.pulse.builder.call`
    within an active building context.     
```

Just added a link to the tutorial so that people can easily migrate.